bibliography/library.bib

```diff
@@ -893,11 +893,11 @@ @inproceedings{venna2001neighborhood
   title         = {Neighborhood Preservation in Nonlinear Projection Methods: An Experimental Study},
   author        = {Jarkko Venna and Samuel Kaski},
   year          = {2001},
-  booktitle     = {Artificial Neural Networks {\textemdash} {ICANN} 2001},
+  booktitle     = {Artificial Neural Networks {---} {ICANN} 2001},
   publisher     = {Springer Berlin Heidelberg},
   pages         = {485--491},
-  doi           = {{10.1007/3-540-44668-0\_68}},
-  url           = {{https://doi.org/10.1007/3-540-44668-0\_68}},
+  doi           = {10.1007/3-540-44668-0_68},
```

Comment:
Problem here is this is no longer valid bibtex...

Reply:
that's what I guessed